modules/apps/commerce/commerce-product-content-web/src/main/java/com/liferay/commerce/product/content/web/internal/helper/CPContentHelperImpl.java

```diff
@@ -480,6 +490,47 @@ public String getImageURL(FileEntry fileEntry, ThemeDisplay themeDisplay)
 		return cpMedia.getURL();
 	}
 
+	@Override
+	public String getIncomingQuantityLabel(
+			long companyId, String sku, HttpServletRequest httpServletRequest)
```

Comment:
We can get the companyId from the request here, No need to pass it as a parameter.